Fluid.Tests/Fluid.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
should probably drop net6.0 all around

Reply:
When I format the file in VS it uses tabs over spaces, regular tab is 4 spaces